src/WebApi.OutputCache.V2/CacheOutputAttribute.cs

```diff
@@ -242,12 +255,12 @@ public override void OnActionExecuting(HttpActionContext actionContext)
                         _webApiCache.Add(baseKey, string.Empty, cacheTime.AbsoluteExpiration);
                         _webApiCache.Add(cachekey, content, cacheTime.AbsoluteExpiration, baseKey);
```

Comment:
so this code would only run if the request wasn't cached in the first place.
I don't think you should invalidate the cache for everyone by any client call, since that would defeat the purpose of caching at all. I could then write a malicious client that just busts all the cache on the server all the time, which is definitely not good